apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -56,6 +64,41 @@ ${context}
 `;
 }
 
+async function checkRender(
+  status: 'success' | 'error',
+  errors: string[],
+  courseId: string,
+  userId: string,
+  questionId: string,
+) {
+  // If there was any issue generating the question, we won't yet check rendering.
+  if (status === 'error' || errors.length > 0) return [];
+
+  const question = await selectQuestionById(questionId);
+  const course = await selectCourseById(courseId);
+  const user = await selectUserById(userId);
+  const authnUser = user; //user id == authn user id in this case
+
+  const locals = {
+    // The URL prefix doesn't matter here since we won't ever show the result to the user.
+    urlPrefix: '',
+    question,
+    course,
+    user,
+    authn_user: authnUser,
```

Comment:
Minor nit:

```suggestion

  const locals = {
    // The URL prefix doesn't matter here since we won't ever show the result to the user.
    urlPrefix: '',
    question,
    course,
    user,
    authn_user: user, // We don't have a separate authn user in this case.
```